simple_network_sim/network_of_populations/__init__.py

```diff
@@ -584,6 +583,8 @@ def _computeInfectiousContactsStochastic(
     numberOfContacts = random_state.poisson(contacts, size=int(infectious))
```

Comment:
Shouldn't we cap the number of contacts before calling poisson?

Reply:
We could do as well, that would lower the samples coming out of Poisson. But we'd still need to cap the outputs of Poisson. as it's just the average the Poisson will be centered around.

Reply:
Ah yeah, I see the problem now.